face/CustomerCheckinManagement/python/customer_checkin_management.ipynb

```diff
@@ -44,51 +47,99 @@
     "\n",
     "image_path = \"path_to_image.jpg\"\n",
     "patient_name = \"Patient Name\" # Optional\n",
```

Comment:
BTW do we want to keep using the term "patient" in this sample? I thought it was for the clinic one?